pkg/process/checks/container_test.go

```diff
@@ -1,9 +1,9 @@
+//go:build linux
```

Comment:
should this be removed? (also ecs/common.go)

Reply:
Newer gofmt's add that automatically as part of transitioning from the old tag format.  We should probably just do it once everywhere, but piecemeal isn't hurting either.

Reply:
this comes from the golang version used.

Reply:
(I'll make a PR to add this everywhere, up shortly..)

Reply:
https://github.com/DataDog/datadog-agent/pull/10601 we'll see how long it takes to get the 113 reviews that requires :)